requirements/requirements-extras-sagemaker-sdk.txt

```diff
@@ -0,0 +1,4 @@
+requests<2.21,>=2.20.0
+sagemaker
+s3fs
+tox
```

Comment:
We should be more specific with version numbers.

Reply:
The thing is, we would realistically always need the newest versions of sagemaker and s3fs, since they are the apis to se and sagemaker. We could put the currently newest versions here?